types/react-date-range/index.d.ts

```diff
@@ -60,9 +60,9 @@ export interface CommonCalendarProps {
 	/** default: none */
 	onChange?: (range: Range) => void;
 	/** default: none */
-	minDate?: DateInputType;
+	minDate?: DateInputType | Date;
```

Comment:
Based on the documentation, wouldn't DateInputType be incorrect as it only states this is a Date?

Reply:
I believe DateInputType is correct for the legacy/depreceated version of the react-date-range package.

As this is my first ever PR, I was aiming for a change least likely to cause problems. Although I appreciate this PR would now mean that anyone using the depreceated version could conclude that Date is now a valid option for them when it isn't.

Reply:
@DanielRosenwasser , I'm not sure what's the best approach for this scenario. Should the d.ts file match the latest version, breaking the deprecated ones or should we create a subfolder for the older version? Do you have any input on this?

Reply:
@AfterEightMint , I did a little more research on this and noticed that the `latest` is 0.9.4 (legacy) and the new type is on `@next`. You know more about this project, if the majority of users are using `react-data-range@next` instead of `@latest` I guess is ok to merge this PR; otherwise I think is better to wait for the new version to be released.

Let me know what you think.